### PR TITLE
Allow choosing a revision to request

### LINKED ISSUE #26: Allow choosing a revision to request
Right now, we show the latest build. We should allow the user to choose a different revision.

### PR DESCRIPTION
**Pull request by vipulchhabra99**
Fixes #26

### CODE REVIEW

File: report.html [hunk 1]
```diff
@@ -5,6 +5,17 @@
 <link rel="stylesheet" href="style.css" type="text/css" media="all" />
 <link rel="stylesheet" href="prism.css" />
 <title>mozilla-central coverage</title>
+<script type="text/javascript">
```
Comment (marco-c) [inline report.html]: Try to re-use the code which we already have to handle options (expanding it if needed), like https://github.com/marco-c/code-coverage-reports/pull/30 did for the `last_push` field.

File: report.html [hunk 2]
```diff
@@ -18,6 +29,10 @@
 <input type="checkbox" name="rust" id="rust" checked="checked"><label for="rust">Rust</label>
 <br />
 <br />
+<form id='revision'>
+    Changest Value : <input type="text" name="changeset-value" id="changeset-value">&nbsp;
```
Comment (marco-c) [inline report.html]: `Changeset:`

File: report.js [hunk 3]
```diff
@@ -65,7 +65,16 @@ async function showFile(file) {
     language = 'java';
   }
 
-  const changeset = await get_latest();
+  if(window.localStorage['text_val']=='')
```
Comment (marco-c) [inline report.js]: We should not use localStorage, it's a synchronous store that has performance problems. In general, we don't really need to store this in a DB, it's a temporary option.

Comment (vipulchhabra99) [reply report.js]: ya but what happens is whenever i submit the values in fields on pressing submit they get disappeared due to which i am unable to put value for changeset

File: report.html (hunk 1 above)
Comment (vipulchhabra99) [reply report.html]: I used for storing value on pressing submit and removing the value on refresh.

Comment (vipulchhabra99) [reply report.html]: hey @marco-c the other options i could think off is using ajax and the other using a cookie to store the value of input column if not the following methods could be adopted please suggest some other method

Comment (vipulchhabra99) [reply report.html]: @marco-c please suggest some methods I should adopt for same so that I can fix the issue.

Comment (marco-c) [reply report.html]: @vipulchhabra99 I already gave you a suggestion. Maybe it's better if you work on a few personal projects first, to gain a bit more experience, and then come back to this once you're ready!

**Pull request closed without merging**